pkg/controller/clustercontroller/backups_test.go

```diff
@@ -139,6 +140,37 @@ var _ = Describe("Test cluster reconciliation queue", func() {
 					return *j.BackupRunning
 				}).Should(Equal(false))
 			})
+
+			It("start jobs to schedule backups", func() {
+
+				j := job{
+					Name:          cluster.Name,
+					Namespace:     cluster.Namespace,
+					myClient:      myClient,
+					BackupRunning: new(bool),
+					lock:          new(sync.Mutex),
+					BackupScheduleJobsHistoryLimit: cluster.Spec.BackupScheduleJobsHistoryLimit,
+				}
+
+				j.Run()
```

Comment:
You can set `cluster.Spec.BackupScheduleJobsHistoryLimit` to a lower value, eg. 2 an write a for loop here to `cluster.Spec.BackupScheduleJobsHistoryLimit + 1`.